source/includes/aws/_buckets.md

```diff
@@ -127,11 +206,15 @@ curl -X DELETE \
 }
 ```
 
-<code>DELETE /services/<a href="#administration-service-connections">:service_code</a>/<a href="#administration-environments">:environment_name</a>/buckets/:regionName/:bucketName</code>
+<code>DELETE /services/<a href="#administration-service-connections">:service_code</a>/<a href="#administration-environments">:environment_name</a>/buckets/:regionName/:bucketName&operation=delete</code>
```

Comment:
is this actually required in the request? i would imagine the DELETE made it implicit that it was a delete operation

Reply:
You're right :) just checked